packages/shared/src/styles/themes.scss

```diff
@@ -15,7 +15,7 @@
     --text-lh-l: 1.75;
     --text-lh-m: 1.5;
     --text-lh-s: 1.25;
-    --text-lh-x: 1.4;
+    --text-lh-xs: 1.4;
```

Comment:
We also have `1.43` in designs. Could we do something like [this](https://github.com/binary-com/deriv-app/pull/1498/files#diff-f027768a0e280a056d3a5738fea3f838caaa9b7f0e55817c8834ca56ca6cc1ce)?

Reply:
I think the differences between `1.4` and `1.43` is probably negligible to the human eye in most cases, let's just stick with `1.4`.